kb/compiling-and-building/compiling-mbedtls-in-mingw.md

```diff
@@ -76,25 +76,15 @@ If you chose not to install Perl or Python, then you can skip building the full
 mingw32-make CC=gcc no_test
 ```
 
-You can then run a much more minimal set of unit tests with:
```

Comment:
The knowledge base applies at least to all supported versions, and preferably also to older versions that we don't officially support but that others (e.g. Linux distributions) do. So we definitely need to keep all information that's relevant to 3.6, and we should keep information about 2.x for a while longer.

Where things are different in different versions of Mbed TLS (and TF-PSA-Crypto), this should be mentioned in the next. For example:

> In versions of Mbed TLS up to 3.x, you can then run …